api/test_data/readmes/SINGLE-CELL_ANN-DATA.md

```diff
@@ -0,0 +1,76 @@
+Generated on: 2024-08-15
+
+# Alex's Lemonade Stand Foundation Single-cell Pediatric Cancer Atlas
+
+The [Single-cell Pediatric Cancer Atlas](https://scpca.alexslemonade.org) is a database of single-cell and single-nuclei data from pediatric cancer clinical samples and xenografts, built by the [Childhood Cancer Data Lab](https://www.ccdatalab.org/) at [Alex's Lemonade Stand Foundation](https://www.alexslemonade.org/).
+
+## Contents
+
+This download includes single-cell or single-nuclei gene expression files and associated metadata for samples from project [SCPCP999990](https://scpca.alexslemonade.org/projects/SCPCP999990) in the ScPCA portal.
```

Comment:
```suggestion
This download includes single-cell or single-nuclei gene expression files and associated metadata for samples from project [TEST_PROJECT_1](https://scpca.alexslemonade.org/projects/TEST_PROJECT_1) in the ScPCA portal.
```

Reply:
Here, I’ve used `PROJECT_ID` as a placeholder in the `test_data/readmes` files and added a `project_id` argument (default: `""`) to `assertProjectReadmeContent` to replace this placeholder with each test's project Id.

Please let me know your thoughts on this, thank you David!